### ISSUE TITLE
Playback rate setting resets when switching events in record view

### ISSUE THREAD

**Issue by haggercody** [blakeblackshear/frigate#1856]
**Describe the bug**
1. go into record view and select an event to play
2. set the playback rate to 8x
3. select another event to play
4. notice the playback rate is back to 1x, but the ui indicates 8x still

**Version of frigate**
Beta 0.90 version 2.3 (HA Addon)

**Config file**
Include your full config file wrapped in triple back ticks.
```
mqtt:
  host: 192.168.10.207
  user: mqtt
  password: "1234"

cameras:

  side:
    ffmpeg:

      input_args:
        - -avoid_negative_ts
        - make_zero
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts
        - -rtsp_transport
        - tcp
        - -stimeout
        - "5000000"
        - -use_wallclock_as_timestamps
        - "1"


      hwaccel_args:
        - -hwaccel
        - vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
  
      inputs:
        - path: rtsp://xxx:xxxx@192.168.10.12:554/cam/realmonitor?channel=1&subtype=1
          roles:
            - detect #blah

        - path: rtsp://xxxx:xxxx@192.168.10.12:554/cam/realmonitor?channel=1&subtype=0
          roles:      
            - record    

    detect:
      width: 640
      height: 480
      fps: 6
      enabled: True

    motion:
      mask: 0,0,640,0,640,310,0,274
      threshold: 20
      contour_area: 100
      delta_alpha: 0.4
      frame_alpha: 0.2
      frame_height: 180

    objects:

      track:
        - person

      filters:
        person:
          min_score: 0.6
          threshold: 0.7
          mask: 640,0,640,288,581,254,116,229,0,244,0,0


    record: 
    # Optional: Enable recording (default: shown below)
      retain_days: 1
      enabled: True   
      events:
        objects:
          - person
          
        pre_capture: 5
        # Optional: Number of seconds after the event to include in the clips (default: shown below)
        post_capture: 5

  front:
    ffmpeg:

      input_args:
        - -avoid_negative_ts
        - make_zero
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts
        - -rtsp_transport
        - tcp
        - -stimeout
        - "5000000"
        - -use_wallclock_as_timestamps
        - "1"

      hwaccel_args:
        - -hwaccel
        - vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
    
      #input_args: -tag:v hvc1
      inputs:
        - path: rtsp://xxxx:xxxx@192.168.10.203:554/Streaming/Channels/101
          roles:
          - detect #blah               
          - record

    zones:
      front_door:
        coordinates: 680,1296,228,1296,43,1097,284,1047

        #input_args: -tag:v hvc1
    detect:
      width: 2304
      height: 1296
      fps: 5
      enabled: True
      max_disappeared: 80

    motion:    
      mask: 2304,0,2304,854,771,736,0,699,0,0
      threshold: 20
      contour_area: 100
      delta_alpha: 0.4
      frame_alpha: 0.2
      frame_height: 180

    objects:

      track:
        - person
        #- dog

      filters:
        person:
          min_score: 0.5
          threshold: 0.7
          mask: 2304,0,2304,741,688,637,0,589,0,0

    record: 
    # Optional: Enable recording (default: shown below)
      retain_days: 1
      enabled: True   
      events:
        objects:
          - person
          #- dog
          
        pre_capture: 5
        # Optional: Number of seconds after the event to include in the clips (default: shown below)
        post_capture: 5


  garage:
    ffmpeg:

      input_args:
        - -avoid_negative_ts
        - make_zero
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts
        - -rtsp_transport
        - tcp
        - -stimeout
        - "5000000"
        - -use_wallclock_as_timestamps
        - "1"

      hwaccel_args:
        - -hwaccel
        - vaapi
        - -hwaccel_device
        - /dev/dri/renderD128

      #input_args: -tag:v hvc1
      inputs:
        - path: rtsp://192.168.10.95/ch1.h264
          roles:
            - detect #blah
          
        - path: rtsp://192.168.10.95/ch0.h264
          roles:           
            - record

      #input_args: -tag:v hvc1
    detect:
      width: 640 
      height: 386
      fps: 6
      enabled: True
    
    motion:
      mask: 640,0,640,103,0,122,0,0  
      threshold: 20
      contour_area: 100
      delta_alpha: 0.4
      frame_alpha: 0.2
      frame_height: 180
    
    objects:
      track:
        - person
        #- dog

      filters:
        person:
          min_score: 0.6
          threshold: 0.7
          mask: 640,0,640,103,0,122,0,0  

    record: 
    # Optional: Enable recording (default: shown below)
      retain_days: 1
      enabled: True   
      events:
        objects:
          - person
          #- dog
          
        pre_capture: 5
        # Optional: Number of seconds after the event to include in the clips (default: shown below)
        post_capture: 5

detectors:
  coral:
    type: edgetpu
    device: usb

```

**Frigate container logs**
```
Include relevant log output here
```

**Frigate stats**
```json
Output from frigate's /api/stats endpoint
```

**Comment by blakeblackshear** [comment]
I noticed this a while back too.

**Comment by hunterjm** [comment]
Fixed in #2775
